src/crimson/common/perf_counters_collection.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "common/perf_counters.h"
+#include <seastar/core/sharded.hh>
+
+namespace ceph::common {
+class PerfCountersCollection: public seastar::sharded<PerfCountersCollection>
```

Comment:
we plan dump perfcounters one seastar core(thread) by one core just like ceph does for messenger worker thread.  each core has its own PerfCountersCollection instance to avoid lock,  and each PerfCountersCollections has its own set of PerfCounters, if register same instance perfCounters to multiple PerCountersCollection will need lock when update perfcounters content, so we do not support it.